.github/workflows/ci.yml

```
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build-with-various-java-version:
+    strategy:
+      matrix:
+        version: [8, 11, 17, 21]
+        project:
+            - 'microservice-transaction-sample'
+            - 'multi-storage-transaction-sample'
+            - 'scalardb-kotlin-sample'
+            - 'scalardb-sample'
+            - 'scalardb-sql-jdbc-sample'
+            - 'spring-data-microservice-transaction-sample'
+            - 'spring-data-multi-storage-transaction-sample'
+            - 'spring-data-sample'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up JDK ${{ matrix.version }}
+      uses: actions/setup-java@v4
+      with:
+        java-version: ${{ matrix.version }}
+        distribution: 'temurin'
+
+    - name: Setup Gradle
+      uses: gradle/actions/setup-gradle@v4
+
+    - name: Build with Gradle Wrapper
+      working-directory: ${{ matrix.project }}
+      run: ./gradlew build
```

```suggestion
      run: ./gradlew build

```
It's a minor thing, but as the GitHub UI shows, I think it would be better to add a new line at the end of the YAML file.